WarehouseManagement.Api/Program.cs

```diff
@@ -20,25 +21,23 @@
 builder.Services.AddApplicationService();
 
 builder.Services.AddControllers();
-builder.Services.AddDbContext<WarehouseManagementDbContext>(options =>
-        options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
+builder.Services.AddDbContext<WarehouseManagementDbContext>(options => 
+        options.UseSqlServer(builder.Configuration["ConnectionStrings:DefaultConnection"]));
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
 var app = builder.Build();
 
-if (app.Environment.IsDevelopment())
+using (var scope = app.Services.CreateScope())
 {
-    using (var scope = app.Services.CreateScope())
-    {
-        var salesContext = scope.ServiceProvider.GetRequiredService<WarehouseManagementDbContext>();
-        salesContext.Database.EnsureCreated();
-    }
-
-    app.UseSwagger();
-    app.UseSwaggerUI();
+    var salesContext = scope.ServiceProvider.GetRequiredService<WarehouseManagementDbContext>();
+    salesContext.Database.EnsureCreated();
```

Comment:
`EnsureCreated()` is not recommended for production scenarios because it bypasses migrations and can lead to an incomplete schema. A better approach is to use `Migrate()` which applies any pending migrations and ensures that the database is in sync with the current model.

Reply:
Looks like to be working, thank you for taking a look! You think now we can merge it?

Reply:
Ok, maybe no. now its failing inside Kubernates

Reply:
Ok, nevermind. i fixed it now. There was a problem because the last db was created using `EnsureCreated` and it was breaking. New db was created with `Migrate` and everything is working!!!

Now back to the first comment i wrote 😄 , is it now good and can we release this?

Reply:
Looks good to me, except the connection string that is exposed in `appsettings.json`. To be more secure, let's move it in `secrets.json`.

Reply:
This is next in my Todo.